components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -2280,6 +2290,130 @@ impl Matrix3D {
     }
 }
 
+/// <https://drafts.csswg.org/css-transforms/#interpolation-of-transforms>
+impl ComputedRotate {
+    fn fill_unspecified(rotate: &ComputedRotate)
```

Comment:
I don't understand the relation between that method name and the spec link just above.